.github/project.yml

```diff
@@ -1,4 +1,4 @@
 release:
-  current-version: 1.0.0
-  next-version: 1.0.1-SNAPSHOT
+  current-version: 1.0.1
+  next-version: 1.0.2-SNAPSHOT
```

Comment:
I'd recommend using `999-SNAPSHOT`, then you'll never need to change the next-version again ;)

Reply:
OK, I get it.